mqtt_cloud_pubsub_proxy/src/main/java/com/google/cloud/pubsub/proxy/gcloud/GcloudPubsub.java

```diff
@@ -95,8 +95,7 @@
   private static final String HASH_PREFIX = "sha256-";
   private static final String PREFIX = "cps-";
   private static final String ASTERISK_URLENCODE_VALUE = "%2A";
-  // TODO - read project name from config file
-  private static final String CLOUD_PUBSUB_PROJECT_NAME = "iot-cloud-pubsub";
+  private static final String CLOUD_PUBSUB_PROJECT_NAME = System.getenv("GCLOUD_PUBSUB_PROJECT_ID");
```

Comment:
So what happens if there's no such environment variable? If the error message is user friendly enough, I'm fine, but if it's mysterious, I'd rather provide an error message saying "Please set the GCLOUD_PUBSUB_PROJECT_ID environment variable to run this program".